device/mellanox/x86_64-mlnx_msn4600-r0/sensors.conf

```diff
@@ -0,0 +1,189 @@
+        label in3 "PMIC-7 ASIC 1.2V Rail_1 T0_3 (out)"
+        label in4 "PMIC-7 ASIC 1.2V Rail_2 T4_7 (out)"
+        label temp1 "PMIC-7 Temp 1"
+        label temp2 "PMIC-7 Temp 2"
+        label power1 "PMIC-7 ASIC 12V Rail_1 Pwr (in)"
+        label power2 "PMIC-7 ASIC 12V Rail_2 Pwr (in)"
+        label power3 "PMIC-7 ASIC 1.2V Rail_1 T0_3 Pwr (out)"
+        label power4 "PMIC-7 ASIC 1.2V Rail_2 T4_7 Pwr (out)"
+        label curr1 "PMIC-7 ASIC 12V Rail_1 Curr (in)"
+        label curr2 "PMIC-7 ASIC 12V Rail_2 Curr (in)"
+        label curr3 "PMIC-7 ASIC 1.2V Rail_1 T0_3 Curr (out)"
+        label curr4 "PMIC-7 ASIC 1.2V Rail_2 T4_7 Curr (out)"
+
+bus "i2c-15" "i2c-1-mux (chan_id 6)"
+    chip "tps53679-i2c-*-58"
+        label in1 "PMIC-8 PSU 12V Rail (in1)"
+        label in2 "PMIC-8 PSU 12V Rail (in2)"
+        label in3 "PMIC-8 COMEX 1.8V Rail (out)"
+        label in4 "PMIC-8 COMEX 1.05V Rail (out)"
+        label temp1 "PMIC-8 Temp 1"
+        label temp2 "PMIC-8 Temp 2"
+        label power1 "PMIC-8 COMEX 1.8V Rail Pwr (out)"
+        label power2 "PMIC-8 COMEX 1.05V Rail Pwr (out)"
+        label curr1 "PMIC-8 COMEX 1.8V Rail Curr (out)"
+        label curr2 "PMIC-8 COMEX 1.05V Rail Curr (out)"
+    chip "tps53679-i2c-*-61"
+        label in1 "PMIC-9 PSU 12V Rail (in1)"
+        label in2 "PMIC-9 PSU 12V Rail (in2)"
+        label in3 "PMIC-9 COMEX 1.2V Rail (out)"
+        ignore in4
+        label temp1 "PMIC-9 Temp 1"
+        label temp2 "PMIC-9 Temp 2"
+        label power1 "PMIC-9 COMEX 1.2V Rail Pwr (out)"
+        ignore power2
+        label curr1 "PMIC-9 COMEX 1.2V Rail Curr (out)"
+        ignore curr2
+
+# Power supplies
+bus "i2c-4" "i2c-1-mux (chan_id 3)"
+    chip "dps460-i2c-*-58"
+        label in1 "PSU-1(L) 220V Rail (in)"
+        ignore in2
+        label in3 "PSU-1(L) 12V Rail (out)"
+        label fan1 "PSU-1(L) Fan 1"
+        ignore fan2
+        ignore fan3
+        label temp1 "PSU-1(L) Temp 1"
+        label temp2 "PSU-1(L) Temp 2"
+        label temp3 "PSU-1(L) Temp 3"
+        label power1 "PSU-1(L) 220V Rail Pwr (in)"
+        label power2 "PSU-1(L) 12V Rail Pwr (out)"
+        label curr1 "PSU-1(L) 220V Rail Curr (in)"
+        label curr2 "PSU-1(L) 12V Rail Curr (out)"
+    chip "dps460-i2c-*-59"
+        label in1 "PSU-2(R) 220V Rail (in)"
+        ignore in2
+        label in3 "PSU-2(R) 12V Rail (out)"
+        label fan1 "PSU-2(R) Fan 1"
+        ignore fan2
+        ignore fan3
+        label temp1 "PSU-2(R) Temp 1"
+        label temp2 "PSU-2(R) Temp 2"
+        label temp3 "PSU-2(R) Temp 3"
+        label power1 "PSU-2(R) 220V Rail Pwr (in)"
+        label power2 "PSU-2(R) 12V Rail Pwr (out)"
+        label curr1 "PSU-2(R) 220V Rail Curr (in)"
+        label curr2 "PSU-2(R) 12V Rail Curr (out)"
+
+# Chassis fans
+chip "mlxreg_fan-isa-*"
+    label fan1 "Chassis Fan Drawer-1"
+    label fan2 "Chassis Fan Drawer-2"
+    label fan3 "Chassis Fan Drawer-3"
```

Comment:
need to recheck this file once platform tests and sensor test is passing. until then we cannot approve it.
BTW i believe you should have asked Vadim P to review it ir the HW guy who worked with Shlomi on it for SN4600C.

Reply:
I talked with Vadim, he didn't have time for doing a full review but told me to check that it the same as the Leopard, I checked and they are only different on the fans, which he said that it is correct.

Reply:
you can manually verify it with running a sensors dump command and then the sensors normal command.
check if the raw data we get and the normal data we get are aligned, I mean you don't have an output with the raw names for some of the components. All should be mapped with a user friendly names for human readable values.
raw output: "sensors -c /dev/null"
mapped output: "sensors"